sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/implementation/util/ModelHelper.java

```diff
@@ -265,6 +265,8 @@ public static BlobItem populateBlobItem(BlobItemInternal blobItemInternal) {
         blobItem.setObjectReplicationSourcePolicies(
             transformObjectReplicationMetadata(blobItemInternal.getObjectReplicationMetadata()));
 
+        blobItem.setHasVersionsOnly(blobItemInternal.isHasVersionsOnly());
```

Comment:
`isHasVersionsOnly` :scream: . The generator's code is very idiomatic.

Reply:
Yeah - it's kinda bad about booleans